core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -1155,6 +1260,8 @@ private void processParse(
 
         short parameterCount = getShort(lo);
 
+        IntList bindVariableTypes = intListPool.next();
```

Comment:
this pool is being cleared when server stops it seems. E.g. this is heap memory leak.